### DISCUSSION TITLE
Question: Can i password-protect the zip-file but not the entries in it?

### DISCUSSION THREAD

**Discussion by HansKindberg** [icsharpcode/SharpZipLib#797] [Q&A]
When I use ZipOutputStream.Password each entry in the created zip-file is password-protected. I expected the zip-file itself to be password-protected. Is it possible to password-protect the zip-file I want to create and not the entries in it?

**Comment by mudnug** [comment]
I believe that you cannot per the ZIP file format specification ([wikipedia](https://en.wikipedia.org/wiki/ZIP_(file_format))).
